src/llvm-late-gc-lowering.cpp

```diff
@@ -1152,12 +1152,12 @@ static SmallSetVector<AllocaInst *, 8> FindSretAllocas(Value* SRetArg) {
                 } else {
                     llvm_dump(SI);
                     dbgs() << "Malformed Select\n";
-                    abort();
+                    return {};
```

Comment:
Why'd we get rid of the aborts here?

Reply:
It just got moved elsewhere, I wanted to print the whole function and that wasn’t available in the function